assets/src/admin/amp-plugin-install.js

```diff
@@ -1,14 +1,17 @@
 /**
  * WordPress dependencies
  */
+/**
+ * External dependencies
+ */
+import { AMP_PLUGINS, AMP_COMPATIBLE } from 'amp-plugins'; // From WP inline script.
+import { debounce } from 'lodash';
```

Comment:
This was moved here via eslint auto-fix. Clearly it didn't do it right. The other instances in this PR are largely wrong too.